PROCESS.md

```diff
@@ -0,0 +1,171 @@
+[] There is general consensus among the core contributors for adding the feature
+  to the spec
+[] Tests are available in the test suite
+[] Two implementations have implemented the feature and pass the tests suite
+[] A blog post has been published introducing the feature
+[] A PR for the spec has been approved that completely specifies the feature and
+  removes the feature from the `STAGE-0` registry.
+
+### STAGE-1
+`STAGE-1` features are included in the specification, but flagged as `STAGE-1`
+and are not subject to the compatibility requirements of "stable" features. They
+can be added at any time as long as it meets all criteria for `STAGE-1` status.
+
+At this stage, the feature should be seeing real world use. These features MAY
+change or be removed all together based on what we learn from real world usage,
+but anything more than small changes should be extremely rare. Any necessary
+feature experimentation and evolution should have been done in `STAGE-0`.
+
+Implementers are encouraged to implement `STAGE-1` features, but are not
+expected to maintain support for previous versions if they change. Users who
+choose to use these features should be prepared for the possibility of changes
+and be willing to update their schemas if a change does occur.
+
+A `STAGE-1` feature can be promoted to `STAGE-2` at any time. Ideally a
+feature should stay in `STAGE-1` for about one year, but may stay longer if we
+don't see it used enough in the wild.
+
+**Promotion to STAGE-2 Checklist:**
+[] There is general consensus that the feature has been proven to be a good
+  addition to JSON Schema and is unlikely to change.
+[] We see the feature being used successfully in the wild and not generating a
+  lot of community support issues that could indicate a problem.
+
+### STAGE-2
+`STAGE-2` features are in the last stages of becoming stable. Changes are
+unlikely, but backward incompatible changes are still allowed in extreme cases.
+Users can be reasonably sure that the feature won't change, but can't expect all
+implementations to support the feature yet.
+
+`STAGE-2` is a staging area for features that are ready to be declared stable,
+but are waiting for the next release or don't yet have enough implementations.
+When we see the vast majority of actively maintained implementations support
+this feature, it will be promoted to stable in the next release. Implementers
+SHOULD support `STAGE-2` features so they aren't out-of-date when the next
+release happens.
+
+Being in `STAGE-2` isn't a guarantee that a feature will be declared stable in
+the next release. If we don't see that the vast majority of actively maintained
+implementation have implemented the feature, it may stay in `STAGE-2` another
+year to allow implementations to catch up.
+
+`STAGE-2` features can only be promoted to stable as part of a release.
+
+**Promotion to Stable Checklist:**
+[] The vast majority of actively maintained implementations support the feature
```

Comment:
> "vast majority"

see https://github.com/json-schema-org/json-schema-spec/pull/1348#discussion_r1024614138

> "actively maintained"

Yep. The more I think about it, the more I agree that this needs more definition. For example, an implementation may be active for bug fixes, but don't intent to add new features. Should that count? This probably needs discussion and probably doesn't need to be decided now as it will be a while before we have to make this determination.

> "support the feature"

The meaning of those words seem pretty clear to me. We could explicitly say it has to pass all the tests. Is there more nuance that I'm missing?

> I think we should investigate further sponsoring implementation work at some point

Sure. I think that speaks to increasing the pool of "actively maintained" implementations.